### PR TITLE
Update Prymont config

### PR DESCRIPTION
**Pull request by terencechain**
Update Prymont config to align https://github.com/protolambda/pyrmont

### CODE REVIEW

**Review by prestonvanloon — APPROVED** [review]

File: shared/params/testnet_pyrmont_config.go [hunk 1]
```diff
@@ -26,7 +26,9 @@ func UsePyrmontConfig() {
 func PyrmontConfig() *BeaconChainConfig {
 	cfg := MainnetConfig().Copy()
 	cfg.MinGenesisTime = 1605700800
+	cfg.GenesisDelay = 432000
 	cfg.NetworkName = "pyrmont"
+	cfg.GenesisForkVersion = []byte{0x00, 0x00, 0x20, 0x09}
 	cfg.SecondsPerETH1Block = 14
```
Comment (prestonvanloon) [inline shared/params/testnet_pyrmont_config.go:32]: This should be 13.

**Review by prestonvanloon — CHANGES_REQUESTED** [review]

File: shared/params/testnet_pyrmont_config.go (hunk 1 above)
Comment (prestonvanloon) [reply shared/params/testnet_pyrmont_config.go:32]: See: ethereum/eth2.0-specs#2132

Comment (prestonvanloon) [reply shared/params/testnet_pyrmont_config.go:32]: Discussed offline. Mainnet is 13, goerli is 14s (or more sometimes)

**Review by prestonvanloon — APPROVED** [review]

**Comment by potuz** [comment]
Don't you want to have 13 seconds per eth1 block as in mainnet?

**Pull request merged**